lms/static/sass/bootstrap/_layouts.scss

```diff
@@ -1,49 +1,82 @@
 // LMS layouts
 
 .content-wrapper {
-    margin-top: $baseline;
-
-    .course-tabs {
-        padding-bottom: 0;
-
-        .nav-item {
-            &.active, &:hover{
-                .nav-link {
-                    border-bottom-color: theme-color("primary");
-                    color: theme-color("primary");
-                }
-            }
-
-            .nav-link {
-                padding: $baseline/2 $baseline*3/4 $baseline*13/20;
-                border-style: solid;
-                border-width: 0 0 $baseline/5 0;
-                border-bottom-color: transparent;
-
-                @include media-breakpoint-down(md) {
-                    border: none;
-                    text-align: left;
-                    padding: 0 0 $baseline/2 0;
-                }
-            }
-        }
-    }
+  margin-top: $baseline;
+  padding-bottom: $baseline*2;
+
+  .course-tabs {
+    padding: 0;
+    font-size: $font-size-sm;
 
-    .main-container {
-        border: 1px solid $border-color;
-        background-color: $body-bg;
+    .nav-item {
+      .nav-link {
+        padding: $baseline/2 $baseline*3/4 $baseline*13/20;
+        border-style: solid;
+        border-width: 0 0 $baseline/5 0;
+        border-bottom-color: transparent;
+        color: theme-color("secondary");
 
-        .page-header {
-            border-bottom: 1px solid $border-color;
-            padding: 20px;
+        @include media-breakpoint-down(sm) {
+          border: none;
+          text-align: left;
+          padding: 0 0 $baseline/2 0;
         }
+      }
 
-        .page-content {
-            padding: 20px;
+      &.active,
+      &:hover {
+        .nav-link {
+          border-bottom-color: theme-color("primary");
+          color: theme-color("primary");
         }
+      }
+    }
+  }
+
+  &.container-fluid {
+    max-width: map-get($container-max-widths, xl);
+  }
+}
+
+.page-header {
+  display: flex;
+  flex-wrap: wrap;
+  border-bottom: 1px solid $border-color;
+  padding: 20px;
+
+  .page-header-main {
+    flex-grow: 1;  // This column should consume all the available space
+  }
+
+  .page-header-secondary {
+    display: flex;
+    margin-left: $baseline;
+
+    &:not(:last-child) {
+      margin-right: $baseline;
+    }
+  }
+}
+
+.page-content-container {
+  border: 1px solid $border-color;
+  background-color: $body-bg;
+}
+
+.page-content {
+  padding: $baseline;
+
+  @include media-breakpoint-up(md) {
+    display: flex;
+
+    .page-content-main {
+      flex-grow: 1;  // This column should consume all the available space
     }
 
-    &.container-fluid {
-        max-width: map-get($container-max-widths, xl);
+    .page-content-secondary {
+      min-width: 350px;
+      max-width: 350px;
```

Comment:
Why not just set ```width: 350px```?

Reply:
That's what I had originally, but it was ending up smaller than 350p. Any ideas why that might be?

Reply:
Was the container set to ```display: flex;```? If so you could try setting ```flex-wrap: wrap'``` on said container.

Reply:
I started out with flex-wrap, but I found that it was wrapping too soon with some browsers. I figured it was better to let the wrapping happen at a breakpoint.

Reply:
I think it would be worth leaving a comment here explaining this, so if someone in the future figures out how to resolve the issue they can update the code with more confidence.

Reply:
Done